src/components/Navbar/index.spec.tsx

```diff
@@ -36,6 +43,30 @@ describe(__filename, () => {
     return store;
   };
 
+  describe('when a version is loaded', () => {
+    it('renders addon name', () => {
+      const addonName = 'addon name example';
+      const store = createStoreWithVersion(
+        {
+          ...fakeVersion,
+          addon: { ...fakeVersionAddon, name: { 'en-US': addonName } },
+        },
+        { makeCurrent: true },
+      );
+      const root = render({ store });
+
+      expect(root.find(`.${styles.addonName}`)).toHaveText(addonName);
+    });
+  });
+
+  describe('when version is undefined', () => {
+    it('does not render addon name', () => {
+      const root = render();
+
+      expect(root.find(`.${styles.addonName}`)).toHaveLength(0);
```

Comment:
The problem is I have `currentVersion===undefined` at the homepage. There looks no easy way to know if it is in loading state or just at the homepage. (I guess we do not want to use `router.location.pathname=='/'` in `Navbar`to handle the homepage as a special case to make it not render `<SkeletonText/>`)

One temporary "fix" would be making [` selectCurrentVersionInfo`](https://github.com/mozilla/addons-code-manager/blob/e64917da1db5bf9714c62b288e0a534103c5d224/src/reducers/versions.tsx#L449) return `null` instead of `undefined` when `currentVersionId===undefined`. Then it renders nothing at the homepage and in an error state. However, I guess, the better way would require introducing something like `compareInfoIsLoading`  in the Store. In which way do you think I should proceed? @kumar303

Reply:
> The problem is I have `currentVersion===undefined` at the homepage. There looks no easy way to know if it is in loading state or just at the homepage. 

@mirefly oh, I see. Whoops. Because of this, you can just make it always render blank (as it does now) -- don't worry about adding skeleton text.

I filed https://github.com/mozilla/addons-code-manager/issues/955 to give us a new *unset* current version state. After that, we could add the skeleton text.

Reply:
I just realized the addon name in Navbar will not be updated to nothing when go back to the homepage. I guess it will be fixed when #955 is done.
STR: 1. Go to homepage; 2. Click 'a browse page'; 3. Click back button in browser. The header still renders addon name 'AndroidADBPlus'. It is expected to render nothing.